app/assets/javascripts/distributions_and_transfers.js

```diff
@@ -94,11 +94,39 @@ $(function() {
     }
   );
 
-  $(function() {
+  $(function () {
     if (storage_location_required && !control.val()) {
       $("#__add_line_item").addClass("disabled");
     }
 
     request_storage_location_and_populate_item(default_item);
   });
+
+  $('#new_distribution').submit(function (e) {
+    e.preventDefault();
+
+    const year = $('select#distribution_issued_at_1i option:selected').val();
+    const month = $('select#distribution_issued_at_2i option:selected').val();
+    const day = $('select#distribution_issued_at_3i option:selected').val();
+
+    if (year && month && day) {
+      let submit;
+      const asked = $(this).data('asked');
+
+      const selectedDate = new Date(year, month - 1, day);
```

Comment:
Why is there a -1 here?

Reply:
Because in javascript the month count starts in zero, like the indexes of an array. It's just a fix to get the correct month.